repooler.py

```diff
@@ -176,139 +257,155 @@ def aggregator(lane_maps,clusters_rem,clusters_per_lane):
     #Ceil(dups) those babies
     raise Exception('Error: Not yet implemented!')
```

Comment:
I've skimmed through this PR. It all looks fine, but since I didn't try to understand what it really does I can't give much feedback. So, it looks good to merge 😎 

At this line you can raise `NotImplementedError`